index.html

```diff
@@ -16,6 +16,7 @@
         <ul>
             <li>An overview of all presentations in reverse chronological order</li>
             <li><a href="intellij">2022-08-03 - IntelliJ</a></li>
+            <li><a href="loadtesting">2022-08-17 - Load testing</a></li>
```

Comment:
Deze moet boven IntelliJ komen. De lijst is bedoeld om de meest recente bovenaan te hebben